test/PositionDescriptor.t.sol

```diff
@@ -51,43 +52,57 @@ contract PositionDescriptorTest is Test, PosmTestSetup {
         vm.snapshotValue("positionDescriptor bytecode size", address(positionDescriptor).code.length);
     }
 
+    function test_bytecodeSize_proxy() public {
+        vm.snapshotValue("proxy bytecode size", address(proxyAsImplementation).code.length);
+    }
+
     function test_setup_succeeds() public view {
-        assertEq(address(positionDescriptor.poolManager()), address(manager));
-        assertEq(positionDescriptor.wrappedNative(), WETH9);
-        assertEq(positionDescriptor.nativeCurrencyLabel(), nativeCurrencyLabel);
+        assertEq(address(proxyAsImplementation.poolManager()), address(manager));
+        assertEq(proxyAsImplementation.wrappedNative(), WETH9);
+        assertEq(proxyAsImplementation.nativeCurrencyLabelBytes(), nativeCurrencyLabelBytes);
+    }
+
+    function test_nativeCurrencyLabel_succeeds() public {
```

Comment:
i added the unit test here!

Reply:
update test to also test the raw bytes32

Reply:
i cant since its private

Reply:
unless we change to public